routes/api.js

```diff
@@ -303,7 +311,7 @@ var getObject = function(req, res, next) {
                 next(err);
             } else {
                 obj.sanitize();
-                res.json(obj);
+                res.json(wantAS2(req) ? as2(obj) : obj);
```

Comment:
I thinks this could be put in a function like "handleAS2" for prevent repeating code (line 547 is the same code)

Reply:
Nice catch, thanks :thumbsup:

Fixed this locally, will push soon